tests/flow/test_memory_usage.py

```
@@ -450,6 +450,22 @@ def test_node_label_overlap_diff_sample_size(self):
         self.graph.query("RETURN 1")
 
+    def test_node_label_overlap_counted_once(self):
+        """test overlapping node attributes are assigned to one label"""
+
+        long_string = 'A' * 1000
+        q = "UNWIND range(0, 4000) AS x CREATE (:A:B:C {v:$long_string})"
+        self.graph.query(q, {'long_string': long_string})
+
+        res = self._graph_memory_usage(20)
+        by_label = dict(zip(res.node_attributes_by_label_storage_sz_mb[0::2],
+                            res.node_attributes_by_label_storage_sz_mb[1::2]))
```

Ruff (0.15.13)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)